config/locales/diaspora/en.yml

```diff
@@ -1327,7 +1334,21 @@ en:
         description: "web+diaspora:// is a new web protocol that we have introduced. Any link to a diaspora* page on an external website that uses this protocol can be opened in the pod on which your diaspora* account is registered. Click the button below to set your browser to use %{pod_url} to recognise external web+diaspora:// links."
         browser: "This protocol is currently in the experimental stage and the success of interactions using it will depend on your browser. If you want to manage or remove this handler, you will do this via your browser settings. The button below will always be enabled, and you need to set the handler separately in each browser you use."
         register: "Register web+diaspora:// handler on this browser"
-
+    
+    import_modal:
+      title: "Import another account"
+      subtext: "You are about to import and merge another account into %{accountname} <br>
+                Here is what will happen:"
+      list1: "Your profile (name, bio, birthday, gender, etc.) will be transferred from the imported account"
+      list2: "Your settings (theme, default post visibility, etc.) will be transferred from the imported account"
+      list3: "All your aspects and contacts from that account will be added to this account"
+      list4: "All posts and comments from that account will be added to this account"
+      closing_text: "The imported account and all its linked data will be deleted from the previous pod.<br>
+                     Your contacts do not need to do anything; they will automatically be connected to this account.<br>
```

Comment:
I would not have the rest of this text in bold – only 'The imported account and all its linked data will be deleted from the previous pod.'

I also suggest separating this text block into three strings, one for each paragraph, rather than using <br>, which I think is not favoured.